### DISCUSSION TITLE
--list-dependencies doesn't seem to work on parallel

### DISCUSSION THREAD

**Discussion by ziima** [tox-dev/tox#3317] [Q&A] [answered]
I was hoping to use `--list-dependencies` to print actually used versions, but it doesn't seem to work on `run-parallel`

```
tox run-parallel --list-dependencies -e quality,py312
py312: OK ✔ in 29.52 seconds
  quality: OK (32.98=setup[32.40]+cmd[0.01,0.35,0.21] seconds)
  py312: OK (29.52=setup[24.55]+cmd[4.97] seconds)
  congratulations :) (33.03 seconds)
```

Do I used it incorrectly?

**Comment by gaborbernat** [comment, marked as answer]
I think this actually works. If you check the log files within the environment, you will see that it does print this information. The only problem here is that this parallel mode actually disables standard output. So you don't see it on the standard output. What I would recommend to do is to add a PR that bypasses the standard output in parallel mode when this flag is set.
